### PR TITLE
SQL: math functions

### PR DESCRIPTION
**Pull request by unrealwork**
- [x] Added tests for math functions.

### CODE REVIEW

**Comment by unrealwork** [comment]
Migrate to TestNG

File: src/test/java/com/axibase/tsd/api/method/sql/function/math/SqlFunctionMathTest.java [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    public static void prepareData() {
+        Series series = new Series(TEST_ENTITY_NAME, TEST_METRIC_NAME);
+        sendSamplesToSeries(series,
+                new Sample("2016-06-29T08:00:00.000Z", "2.11"),
+                new Sample("2016-06-29T08:00:01.000Z", "7.567"),
+                new Sample("2016-06-29T08:00:02.000Z", "-1.23")
+        );
+    }
+    /*
+    Following tasks related to #3039 issue
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/function/math/SqlFunctionMathTest.java]: `tests`? `3049`?

**Comment by Sild** [comment]
review done

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**